aspnetcore-webapi/RequestBodyRead/RequestBodyRead/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+using System.Diagnostics.Contracts;
+
+namespace RequestBodyRead;
```

Comment:
This looks a bit odd having the namespace between the using statements. We should move this to the bottom of the block.